src/plugins/stake-plugins/types.ts

```diff
@@ -77,7 +77,7 @@ export interface StakePolicy {
 // Change Quote
 // -----------------------------------------------------------------------------
 export interface ChangeQuoteRequest {
-  action: 'stake' | 'unstake' | 'claim'
+  action: 'stake' | 'unstake' | 'claim' | 'unstakeexact'
```

Comment:
'unstake-exact' or 'unstakeExact' to keep it consistent with other enumeration string naming.